addons/web/static/src/scss/bootstrap_review_frontend.scss

```diff
@@ -167,3 +167,13 @@ $o-navbar-nav-pills-link-border-radius: $nav-pills-border-radius !default;
     // on the 'currentColor' of the inputs.
     color: inherit;
 }
+
+.form-select {
+    background-clip: padding-box;
```

Comment:
I'd suggest to add a note explaining that's necessary in order to achieve a consistent "border rendering behaviour" across `form-select` and  `form-control`, especially when using semi-transparent borders.